Problem:
youtube certification require this feature, audio clip happened
when insert AD video without recreate new starboard player and
keep AVsync not broken

Solution:
audio decoder drop raw audio data according to clip meta data

---
 ext/opus/gstopusdec.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/ext/opus/gstopusdec.c b/ext/opus/gstopusdec.c
index 7ff9bee..9f4a198 100644
--- a/ext/opus/gstopusdec.c
+++ b/ext/opus/gstopusdec.c
@@ -695,6 +695,11 @@ opus_dec_chain_parse_data (GstOpusDec * dec, GstBuffer * buffer)
 
   cmeta = gst_buffer_get_audio_clipping_meta (buf);
 
+  if (cmeta && cmeta->format == GST_FORMAT_TIME) {
+    cmeta->format = GST_FORMAT_DEFAULT;
+    cmeta->start = gst_util_uint64_scale (cmeta->start, dec->sample_rate, GST_SECOND);
+    cmeta->end = gst_util_uint64_scale (cmeta->end, dec->sample_rate, GST_SECOND);
+  }
   g_assert (!cmeta || cmeta->format == GST_FORMAT_DEFAULT);
 
   /* Skip any samples that need skipping */
-- 
2.25.1
